tekton/pipeline_run_test.go

```diff
@@ -262,4 +269,48 @@ var _ = Describe("PipelineRun", func() {
 			})
 		})
 	})
+
+	Context("When calling getPipelineRef", func() {
+		It("should return a PipelineRef without resolver if the releaseStrategy does not contain a bundle", func() {
+			releaseStrategy := &v1alpha1.ReleaseStrategy{
+				Spec: v1alpha1.ReleaseStrategySpec{
+					Pipeline: "release-pipeline",
```

Comment:
policy is required. One too many deletes :slightly_smiling_face:

Reply:
Not sure how it didn't raise a failure tbh

Reply:
Already! :)